src/workldr/exec_wasmtime/mod.rs

```diff
@@ -25,7 +25,7 @@ pub(crate) mod test {
         let wasmldr = Box::new(Wasmldr);
         assert_eq!(
             wasmldr.exec(),
-            include_bytes!(env!("CARGO_BIN_FILE_ENARX_EXEC_WASMTIME"))
+            include_bytes!(env!("CARGO_BIN_FILE_ENARX_EXEC_WASMTIME_BIN"))
```

Comment:
It occurs to me that these tests are meaningless, I cannot imagine any problem that they would actually detect. But maybe that's for a different PR to fix.